src/gnssro/gnssro_bufr2ioda.f90

```diff
@@ -301,7 +301,7 @@ program gnssro_bufr2ioda2
        gnssro_data%msl_alt(ndata)  = height
        gnssro_data%bend_ang(ndata)     = bend
        gnssro_data%bndoe_gsi(ndata)    = bend_error
-       gnssro_data%impact_para(ndata)  = impact! 
+       gnssro_data%impact_para(ndata)  = impact!
```

Comment:
remove `!` as well?

Reply:
Sure! The space has been removed automatically by my text editor. I just removed it by hand.